Tests iOS/Support/RequestMatchers.swift

```diff
@@ -132,6 +136,14 @@ struct ClientAPIRequest {
 extension ClientAPIRequest {
 
     var itemIdVariable: String {
-        self.apolloRequestBody!.variableDict["itemId"] as! String
+        self.apolloRequestBody!.variableDict["itemID"] as! String
```

Comment:
itemId?

Reply:
Nope, its itemID in the graph when encoded. https://github.com/Pocket/pocket-ios/blob/develop/PocketKit/Sources/PocketGraph/Operations/Mutations/FavoriteItemMutation.graphql.swift#L11

Reply:
actually. i guess this would be in the input dict too, but as id. 🤔 i'll look at this in a followup.